Dynamic-Programming/tests/LevenshteinDistance.test.js

```diff
@@ -0,0 +1,19 @@
+import { calculateLevenshteinDp } from '../LevenshteinDistance'
+
+test('Add and remove letters at the beginning and the end while making replacements', () => {
+  const from = 'kitten'
+  const to = 'sitting'
+  expect(calculateLevenshteinDp(from, to)).toBe(3)
+})
+
+test('Replace contiguous letters at the middle of the words', () => {
+  const from = 'book'
+  const to = 'back'
+  expect(calculateLevenshteinDp(from, to)).toBe(2)
+})
+
+test('Compare properly strings with different length', () => {
```

Comment:
this test is redundant, the first test already has different sizes

Reply:
I believe the more tests the better. Sometimes we see more specific test cases not passing surprisingly.
Simple test cases give us confidence while reviewing the complex ones.

How would you describe this one?

Reply:
Okayy ! 

I would put something like :  "Should return distance when strings length are different"

Reply:
"return" because you are testing a function result

Reply:
Thanks